README.md

```diff
@@ -20,7 +20,7 @@ Just outside their doors is a fleet of experts and technologists with the talent
 
 Seattle is world-renowned as a hub for technology talent, full of motivated builders, designers, and creators that want to use their skills for good. Digital Aid Seattle is bringing these communities together to create a sum greater than its parts.
 
-This website is for organizations that want to [learn more about partnering with Digital Aid Seattle](https://www.digitalaidseattle.org/partner) and for passionate, skilled Seattlites who want to [learn more about volunteering!](https://www.digitalaidseattle.org/volunteer)
+This website is for organizations that want to [learn more about partnering with Digital Aid Seattle](https://www.digitalaidseattle.org/partners) and for passionate, skilled Seattlites who want to [learn more about volunteering!](https://www.digitalaidseattle.org/volunteers)
 
 ## Technology Stack
```

Comment:
#29 @jnakaso https://airtable.com/developers/docs - this one is not working. I suggest updating to https://airtable.com/developers.

Reply:
yes, make that change.